index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Asad Shareef Protfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="container">
+    <header>
+        <a href="#">My Logo</a>
+        <ul>
+            <li><a href="#">Protfolio</a> </li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <nav class="flex-nav"><button>☰</button></nav>
+    </header>
+    <div class="headline">
+        <h1>I’m Asad Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="social-title"><a href="#">LET’S CONNECT</a></p>
```

Comment:
- [ ]  [OPTIONAL ] Kindly   use the` <p> `tag only for paragraph  LET'S CONNECT is not a paragraph please do not use `<p> ` tag for that :+1:

Reply:
I have change p tag in LET'S CONNECT as well